packages/transport-http/src/http-request.js

```diff
@@ -97,17 +98,34 @@ export async function httpRequest({
     }
   }
 
+  function showAccessNodeErrorMessage() {
+    logger.log({
+      title: "Access Node Error",
+      message: `The provided access node ${hostname} does not appear to be a valid REST/HTTP access node.
+Please verify that you are not unintentionally using a GRPC access node.
+See more here: https://docs.onflow.org/fcl/reference/sdk-guidelines/#connect`,
+      level: logger.LEVELS.error,
+    })
+  }
+
   function makeRequest() {
     if (fetchTransport) {
       return fetchTransport(`${hostname}${path}`, {
         method: method,
         body: body ? JSON.stringify(body) : undefined,
+        mode: "no-cors",
```

Comment:
This is failing and triggering the `showAccessNodeErrorMessage` even with correct accessNode.api. Might be addition of `mode: "no-cors"` and removal of `async res`

Reply:
Ok I've got it fixed now.  I was trying to swallow the CORS error generated by Chrome/browser but I guess there is no way to do that.  Best solution I can get is that it will still print out a CORS error but it will print out our error on top.